configuration/configuration.pl

```diff
@@ -1,5 +1,5 @@
 {
-    projectName => 'sw360',
+    projectName => 'sw360_lr7',
```

Comment:
That is not the correct usage for `projectName`. One would use `projectName` to have multiple parallel installations on the same host or in the same cluster (e.g. dev and prod). The name of the liferay zip should be hardcoded and not depend on that string.

Reply:
This was not clear to me due to the naming of "project". Changed.

Reply:
the naming is based on the convention of docker-compose, which uses `COMPOSE_PROJECT_NAME` as scope for deployments.